www/hugo.toml

```diff
@@ -14,18 +14,18 @@ allow = ["pandoc"]
 defaultMarkdownHandler = "pandoc"
 
 [module]
-    [[module.mounts]]
-        source = "../code"
-        target = "assets/js"
-    [[module.mounts]]
-        source = "assets"
-        target = "assets"
-    [[module.mounts]]
-        source = "../corpus"
-        target = "static/corpus"
-    [[module.mounts]]
-        source = "../layouts"
-        target = "static/layouts"
-    [[module.mounts]]
-        source = "static"
-        target = "static"
+  [[module.mounts]]
+    source = "../code"
+    target = "assets/js"
+  [[module.mounts]]
+    source = "assets"
+    target = "assets"
+  [[module.mounts]]
+    source = "../corpus"
+    target = "static/corpus"
+  [[module.mounts]]
+    source = "../layouts"
+    target = "static/layouts"
+  [[module.mounts]]
+    source = "static"
+    target = "static"
```

Comment:
It’s generally not a good idea to mix two subjects in one PR, but since it’s there…
- do we really need the `static` and `assets` mounts ? aren’t they default ?
- why change the indentation ? maybe a zero-indent would be just as good ?

Reply:
Zero-indent would be kind of confusing, if we later have a more complex config. I switched from 4 spaces to 2 because I originally tried to implement the palette in `hugo.toml` (in `[params.theme.{dark,light}]` blocks), before realizing half way through that it was *really stupid*. I can revert the change if you want.

I have no idea why, but removing the `static` and `assets` mounts completely breaks the site. My guess is that they are mounted by default if we don’t specify anything, but mounts *exactly* what we tell it to if we do specify something.

Reply:
My rule of thumb in such cases is that we shouldn’t alter the `git blame` without adding logical value.

Indenting TOML is very debatable, but if we keep the indentation then I’d suggest keeping it as is. Two or for spaces won’t make buch of a difference.